src/modules/common/components/Selector.js

```diff
@@ -49,19 +43,34 @@ export const Wrapper = styled.View`
   `}
 `
 
+type PropsType = {
+  verticalLayout: boolean,
+  closeDropDownCallback?: () => void,
+  items: Array<SelectorItemModel>,
+  selectedItemCode: string | null
+}
+
 /**
  * Displays a Selector showing different items
  */
 class Selector extends React.Component<PropsType> {
   getItems (): React.Node {
-    const {items, activeItemCode} = this.props
+    const {items, selectedItemCode} = this.props
     return items.map(item => {
+      const isSelected = item.code === selectedItemCode
+      if (item.active || isSelected) {
+        return (
+          <TouchTarget key={item.code} onPress={item.onPress}>
+            <ActiveElement selected={isSelected}>
```

Comment:
Is the `theme` passed automatically?

Reply:
Theme should be passed explicitly so this is easier to test.

Reply:
To which components does the theme has to be passed explicitly?

Reply:
Basically every component should have the `theme` in the props. Only containers should be able to get the theme by using `withTheme`

Reply:
Even if they are not "real" react components but just styled components?

Reply:
Imo yes as it makes testing easier and is not much work. At least I would just change the code and pass the theme prop instead of making the test more complicated.